src/Domain/CampaignRepository.php

```diff
@@ -569,7 +606,8 @@ public function search(
         $qb = $this->getEntityManager()->createQueryBuilder();
 
         $safeSortField = match ($sortField) {
-            'matchFundsRemaining' => throw new \Exception('Sorting by matchFundsRemaining not yet implemented'),
```

Comment:
what's the reason for deleting `matchFundsRemaining` here? Don't we still want that as a form of to-do comment to remember that we have to implement that as its expected by the FE?

Reply:
I think I was thinking that `matchFundsUsed` with both sort directions covers the same thing, which is not quite right. I'll think a bit more about how to deal with that – it feels a bit messy to add another field to the stats but perhaps it's a false economy to avoid that redundancy, when setters can guarantee that things add up.